instructor/common.py

```diff
@@ -256,9 +255,18 @@ def assignments_variables(request):
     var_fields = ['has_concentration', 'has_temperature', 'has_start_time',
                   'has_duration', 'has_collection_time']
     AssignmentForm = modelform_factory(models.Assignment, fields=var_fields)
+
+    # Want to know if Treatments were already created
+    treatments_created = False
+    if models.Treatment.objects.filter(assignment=assignment).count() > 0:
```

Comment:
You are right. I thought `exists()` is used only to test the existence of a particular object in the QuerySet.